Program.cs

```diff
@@ -4,15 +4,18 @@
 // Start Process
 var processStartInfo = new ProcessStartInfo
 {
-	FileName = @"BF1942.exe"
+    FileName = @"BF1942.exe"
 };
+
 foreach (var arg in args)
 {
-	processStartInfo.ArgumentList.Add(arg);
+    processStartInfo.ArgumentList.Add(arg);
 }
+
 var process = Process.Start(processStartInfo)!;
+
 var keepAlive = true;
-Console.WriteLine($"Process has started ({process.Id})");
+Console.WriteLine($"[Borderless1942]: [{DateTime.Now:MM/dd/yyyy - hh:mm:ss tt}] [BF1942 Process Has Started]: ({process.Id})");
```

Comment:
I like the more detailed log messages and am generally happy with them. One thing I would switch up is the date format.

Because different countries have different date format rules, I find it best to use `yyyy-MM-dd` as it removes ambiguity between this program running on your computer and it running on mine. (In case you're curious, `yyyy-MM-dd` is part of a standard called ISO 8601)

Reply:
That is definitely a good point regarding the date formatting.  Thank you for pointing out the ISO 8601 standard.  I will keep that in mind for any future contributions or projects that I write code for.  Unfortunately, I seem to be stuck in a mindset of using the MM/dd/yyyy format since that is the format that I have used for my of my career so far.  I need to train myself to think more about the end user experience and that they may not be using the same regional settings that I am.